kolibri/plugins/device/assets/src/views/DeviceSettingsPage/index.vue

```diff
@@ -89,7 +89,7 @@
 
 <script>
 
-  import mapValues from 'lodash/map';
+  import { map } from 'lodash';
```

Comment:
We don't currently have the `sideEffects` webpack 4 optimization turned on, so [in order to support tree shaking](https://stackoverflow.com/questions/35250500/correct-way-to-import-lodash), we need to continue import from `lodash/map`

However, renaming `mapValues` to `map` seems like a good idea

Reply:
That makes sense @indirectlylit. Updated!